src/main/navigation/navigation.c

```diff
@@ -2848,7 +2854,13 @@ void resetLandingDetector(void)
 
 bool isFlightDetected(void)
 {
+#if !defined(USE_VARIABLE_PITCH)
     return STATE(AIRPLANE) ? isFixedWingFlying() : isMulticopterFlying();
+#else
```

Comment:
Same as above,

expand the statement into if else.
if(airplane) else if(helicopter) else { mc }

Reply:
Same reply as above.